tests/unit/models/gpflux/test_models.py

```diff
@@ -249,3 +254,39 @@ def scheduler(epoch: int, lr: float) -> float:
     model.optimize(dataset)
 
     npt.assert_allclose(model.model_keras.optimizer.lr.numpy(), init_lr, rtol=1e-6)
+
+
+def test_deep_gaussian_process_default_optimizer_is_correct(
```

Comment:
Might be good to run the same test for a dummy subclass of DeepGaussianProcess too?

Reply:
done